.github/workflows/e2e-nightly-main.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Build
         working-directory: test/e2e
         # Run make jobs in parallel, since we can't run steps in parallel.
-        run: make -j2 docker generator runner tests
```

Comment:
???

Reply:
tests are built when the runner issues "go test ...." in the call to `exec.CommandVerbose`